docs/manage/configuration.md

```diff
@@ -21,11 +21,34 @@ The credentials look like this:
 "secret":"ab0cd0efghijkl0m12no3pqrstuvw0xyza4bcd567891abcd"}}
 ```
 
-When you turn on your robot, `viam-server` starts up and uses the credentials in its local <file>/etc/viam.json</file> to fetch its full config from the [Viam app](https://app.viam.com).
+When you turn on your robot, `viam-server` starts up and, by default, it uses the credentials in its local <file>/etc/viam.json</file> to fetch its full config from the [Viam app](https://app.viam.com).
 Once the robot has a configuration, it caches it locally and can use the configuration for up to 60 days.
 The robot checks for new configurations every 15 seconds and changes its configuration automatically when a new configuration is available.
 All communication happens securely over HTTPS using secret tokens that are in a robot's configuration.
 
+{{< alert title="Tip" color="tip" >}}
+You can also store your config file in another location.
+To use such a config file, pass in the path to the config when you start `viam-server`:
+
+{{< tabs >}}
+{{% tab name="Linux" %}}
+
+```sh {id="terminal-prompt" class="command-line" data-prompt="$"}
+sudo viam-server -config <path-to-config>.json

```

Comment:
Does this actually start the server? Or does this change where it'll look for its config for ever more in the future?

Reply:
it just starts the server once with that config